website/content/docs/upgrade/v1_11.mdx

```diff
@@ -0,0 +1,87 @@
+
+* Source relaxation: plugins that are available on non-Github hosts may now be installed through the `packer plugins install --path` command.
+  This will install the plugin in the Packer plugin directory, creating the file hierarchy that matches this source.
+  This means you will now be able to use `required_plugins` in your templates with non-Github sources, and have Packer discover it as it would with Github sources.
+  This enables you to add version constraints on those plugins if you so choose.
+  Please note though that while you can add them to `required_plugins` for validation purposes, remote installation is still limited to Github-sourced plugins.
+
+## Troubleshooting
+
+This section provides guidance on potential issues you may experience after upgrading.
+
+### Missing plugins error
+
+Packer may report that you one or more plugins are missing after the upgrade.
+This might happen if you had installed the plugin using one of the now unsupported methods of installation:
+
+* in the directory Packer is invoked from
+* alongside the Packer binary
+* under the root of the `PACKER_PLUGIN_PATH` directory
+
+For those missing components/undiscovered plugins, you may need to reinstall them as described in [Installing Plugins](/packer/docs/plugins/install).
+
+### Plugin local installation
+
+If you cannot remotely install a plugin for any reason (firewalls, Github blocked, etc.), you can use the local installation methods.
+Once you have gotten the plugin binary, you can then install it for Packer to use through the `packer plugins install --path` command.
+This will place the plugin binary in the location that Packer expects it to be, with the correct naming scheme and sumfile, so Packer will be
+able to load it, even with a `required_plugins` block in a template.
+
+### Using `required_plugins`
+
+In some cases, Packer will hint at using `required_plugins` block in your template.
+With such a block, this opens the possibility to use `packer init` to automatically install plugins, and check that your environment is ready to build your template.
+While this is the preferred way to manage the plugins you need in order to build your images, this is not a hard requirement.
+You can still install plugins independently of a template by using the `packer plugins install` command variants.
+If you are using HCL for your templates, the choice belongs to you, while if you are using JSON, that alternative will be what you need to use.
+
+### Installing plugins in one shared directory
+
+This section is mostly split in two use-cases:
+
+1. Installation of all the plugins as `packer-plugin-<name>` under a single root directory
+2. Installation of plugins with the convention described above.
+
+If you are looking for alternative 1, this is unfortunately not supported anymore.
+The reason being that this method was conflicting with `required_plugins`, so we decided to consolidate how loading plugins is done around this method.
```

Comment:
I find the mention of the second option confusion as we don't actually talk about it.

That said, I reword to call out why we made the change. What do you think?

```suggestion
### Installing plugins outside of namespace directories

Previously Packer supported the use of plugins as `packer-plugin-<name>` under a single root directory. In Packer 1.11 this functionality was removed to enhance support for using non-GitHub and development prerelease plugin binaries within required_plugins block.
```